src/app/character-tile/CharacterHeaderXP.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import clsx from 'clsx';
+import { t } from 'app/i18next-t';
+
+import { D1Store } from '../inventory/store-types';
+import PressTip from '../dim-ui/PressTip';
+import { percent } from '../shell/filters';
+
+function getLevelBar(store: D1Store) {
+  if (store.percentToNextLevel) {
+    return {
+      levelBar: store.percentToNextLevel,
+      xpTillMote: undefined,
+    };
+  }
+  if (store.progression?.progressions) {
+    const prestige = store.progression.progressions.find((p) => p.progressionHash === 2030054750); // some kind of d1 hash
+    if (prestige) {
+      const data = {
+        level: prestige.level,
+        exp: prestige.nextLevelAt - prestige.progressToNextLevel,
+      };
+      return {
+        xpTillMote: t('Stats.Prestige', data),
+        levelBar: prestige.progressToNextLevel / prestige.nextLevelAt,
+      };
+    }
+  }
+  return {
+    levelBar: 0,
+    xpTillMote: undefined,
+  };
+}
```

Comment:
```suggestion
function getLevelBar(store: D1Store) {
    const prestige = store.progression?.progressions.find((p) => p.progressionHash === 2030054750); // some kind of d1 hash
    const data = {
      level: prestige?.level,
      exp: prestige?.nextLevelAt - prestige?.progressToNextLevel,
      };
  return {
    levelBar: store.percentToNextLevel ? store.percentToNextLevel : prestige?.level ? (prestige.progressToNextLevel / prestige.nextLevelAt) : 0,
    xpTillMote: store.percentToNextLevel ? '' : prestige?.level ? t('Stats.Prestige', data) : '',
  };
```
```

Reply:
2030054750 is DestinyProgressionDefinition
```
    "2030054750": {
      "progressionHash": 2030054750,
      "name": "character_prestige",
      "scope": 1,
      "repeatLastStep": true,
      "icon": "/common/destiny_content/icons/b2d9054cc06b8334babbe62db77d52b7.png",
      "identifier": "PROGRESSION_CHARACTER_PRESTIGE",
      "steps": [
        {
          "progressTotal": 25000,
          "rewardItems": []
        }
      ],
      "visible": true,
      "hash": 2030054750,
      "index": 0,
      "redacted": false
    },
```

Reply:
oo, nice! thanks for cleaning that up.